pkgs/development/python-modules/pysvn/default.nix

```
@@ -1,5 +1,6 @@
 { stdenv
 , buildPythonPackage
+, pythonPackages
 , fetchurl
 , pkgs
```

```suggestion
, bash
, subversion
, apr
, aprutil
, expat
, neon
, openssl 
```
and thin in `python-packages.nix`, just do:
```
  inherit (pkgs) bash subversion apr aprutil expat neon openssl;
```
to avoid python packages of the same name getting imported.